docs/File-System-API.md

```diff
@@ -58,14 +58,69 @@ OutStreamOptions options = new OutStreamOptions.Builder(ClientContext.getConf())
 FileOutStream out = tfs.getOutputStream(path, options);
 ```
 
+### IO Options
+
+There are two different types of storage in Tachyon, natively managed storage and under storage.
```

Comment:
Tachyon uses two different storage types: natively managed storage and under storage.